aepp/flowservice.py

```diff
@@ -1296,6 +1298,22 @@ def getLandingZoneContainerName(
         """
         return self.getLandingZoneContainer(dlz_type=dlz_type)["containerName"]
 
+    def getLandingZoneStorageName(
```

Comment:
The function name is duplicated. But I think it's a bit too generic, how about something like `getLandingZoneNamespace` where for Azure the "namespace" is the container name and for S3 the "namespace" is the top-level folder?

Reply:
the azure's storageAccountName is bucketName on the AWS

i like your suggestion on namespace, so lets have it as:
getLandingZoneNamespace -> (storageAccountName for azure| bucketName for aws)
and 
getLandingZoneStorageName -> (containerName for azure | dlzFolder for aws)

wdyt?

Reply:
I also like this approach, can you also document this differenciation in the docstring ? 
So people can access it "in-code" ?

Reply:
I just saw you already did below.